### DISCUSSION TITLE
uws unresponsive when sending too many requests.

### DISCUSSION THREAD

**Discussion by TwinFuture** [uNetworking/uWebSockets.js#113] [Q&A] [answered]
I did a simple test using a string which is 22kb, using res.end('22kb string here'). I ran a stresstest trying to basically hit it as hard as I can maximum requests possible and uws just consumes 100% and crashes, resulting in zero requests being processed it seems to just instantly shut it down. I have tested both on nginx and nodes native http and they don't crash and still accept the maximum they can.

41k requests in 69.21s, 910 MB read
515k errors (513k timeouts)

I get this errors also
Error: Returning from a request handler without responding or attaching an abort handler is forbidden!
terminate called without an active exception

I used autocannon for the test. autocannon -c 200 -d 80 -p 200 http://xxx.xxx.xxx.xxx:9001

**Comment by ghost** [comment, marked as answer]
#45
